tests/monitoring_test.go

```diff
@@ -247,10 +248,20 @@ var _ = Describe("Prometheus Alerts", func() {
 		var createResources = func(createDataVolume bool, rxbounceEnabled bool) string {
 			vmName := fmt.Sprintf("testvmi-%v", rand.String(10))
 
+			vmi := NewMinimalVMIWithNS(strategy.GetNamespace(), vmName)
+			vm = NewVirtualMachine(vmi)
+			vm.Spec.Template.ObjectMeta.Annotations = map[string]string{
+				"vm.kubevirt.io/os": "windows-10",
+			}
+			vm.Spec.Running = ptr.To(true)
```

Comment:
+1 because there is a plan to deprecate `vm.Spec.Running`.

Reply:
changed

Reply:
tests are failing because `NewVirtualMachine` is setting the `vm.Spec.Running` to false under the hood, and when I set the `RunStrategy` it errors with 

```
Type: "FieldValueInvalid",
                          Message: "Running and RunStrategy are mutually exclusive",
                          Field: "spec.running",
```

@jcanocan @codingben

Reply:
Can you change the `NewVirtualMachine`, so it does not set the `running` field? I think if we leave it unset, it will be the same as `false`.

Reply:
changed in NewVirtualMachine

```
- Running: &running,
+ RunStrategy: ptr.To(kubevirtv1.RunStrategyHalted),
```